nixos/modules/services/monitoring/scrutiny.nix

```diff
@@ -0,0 +1,216 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.scrutiny;
+
+  # Helper function for inserting null values in place of empty strings in conf file.
+  optionalStr = e: if e != "" then e else null;
```

Comment:
I think we don't need this if you use `lib.types.nullOr` for the options below?

Reply:
Oh nice, I didn't know that. Fixed :+1: